### ISSUE TITLE
Reproducing the results for MD-17 and QM9

### ISSUE THREAD

**Issue by kavi9030** [nicoleyghu/conformal_prediction_in_latent_space#2]
Hello Nicole,

I'm trying to reproduce Figure 3 and Figure S5 for the UQ method comparison on MD17 and QM9 dataset, however I'm unable to match to the published results. Is it possible to get more details of the code setup, parameters and the dataset you used?

Thank you in advance.

**Comment by nicoleyghu** [comment]
Hello, 

Do you have specific methods where the results differ? The conformal prediction UQ should be relatively insensitive to the choice of hyperparameters, but the setup is included in the published Jupyter Notebooks such as: 

`per_calib=0.1 # percent of calibration data
seed=1 # random seed for data split
num_nearest_neighbors=10 # distance calculation for nearest neighbors
metric = "minkowski" # type of distance metric` 

How is the underlying force field/interatomic potential training? What model do you use for the energy prediction? 

MD17-Aspirin dataset is downloaded from http://www.sgdml.org/#datasets, and the full QM9 dataset with the formation energy. 

Best,
Nicole

**Comment by kavi9030** [comment]
Thank you for the prompt response, Nicole. 
None of the methods so that's why I assumed it is coming from the model training.
I'm using the exact same code example you have provided under the `train` folder to train a model with GMP featurization and SingleNN neural network architecture with the MD17-Aspirin dataset.

**Comment by nicoleyghu** [comment]
Do you mind sharing the error metrics you have during training? 

The setup I had includes hyperparameters below:

nsigmas = 10 # in log scale
MCSHs_index = 3
rs = 1.0
NN_index = 32
isSH = True
seed = 1
activation_fx = 'relu'

**Comment by kavi9030** [comment]
I used the same hyperparameters. My Training Energy MAE is 2.4629 kcal/mol and Test Energy MAE is 5.1406 kcal/mol. 
[singleNN_MD17.pdf](https://github.com/nicoleyghu/conformal_prediction_in_latent_space/files/15062211/singleNN_MD17.pdf)

```
  epoch    train_energy_mae    train_loss    val_energy_mae    valid_loss    cp      lr     dur
-------  ------------------  ------------  ----------------  ------------  ----  ------  ------
   1500              2.5275        0.2123            2.5976        0.2182        0.0000  9.2023
   1501              2.5273        0.2123            2.5969        0.2181        0.0000  9.1494
   1502              2.5289        0.2124            2.6073        0.2190        0.0000  9.1477
   1503              2.5273        0.2123            2.5972        0.2182        0.0000  9.1352
   1504              2.5305        0.2126            2.5967        0.2181        0.0000  9.1717
   1505              2.5244        0.2121            2.5982        0.2183        0.0000  9.1539
   1506              2.5239        0.2120            2.5958        0.2181        0.0000  9.1626
   1507              2.5235        0.2120            2.5984        0.2183        0.0000  9.1493
   1508              2.5304        0.2126            2.5987        0.2183        0.0000  9.1614
   1509              2.5230        0.2120            2.5974        0.2182        0.0000  9.1482
   1510              2.5281        0.2124            2.5967        0.2182        0.0000  9.1569
   1511              2.5269        0.2123            2.5981        0.2182        0.0000  9.1327
   1512              2.5289        0.2125            2.6061        0.2189        0.0000  9.1837
   1513              2.5270        0.2123            2.6024        0.2186        0.0000  9.1662
   1514              2.5279        0.2124            2.5974        0.2182        0.0000  9.1715
   1515              2.5275        0.2123            2.6004        0.2184        0.0000  9.1521
   1516              2.5248        0.2121            2.5957        0.2181        0.0000  9.1955
   1517              2.5265        0.2122            2.5972        0.2182        0.0000  9.1644
   1518              2.5290        0.2125            2.5999        0.2184        0.0000  9.1959
   1519              2.5263        0.2122            2.6050        0.2188        0.0000  9.2043
   1520              2.5268        0.2123            2.5987        0.2183        0.0000  9.1908
   1521              2.5258        0.2122            2.5993        0.2184        0.0000  9.1687
   1522              2.5264        0.2122            2.5979        0.2182        0.0000  9.1696
   1523              2.5243        0.2121            2.5975        0.2182        0.0000  9.1741
   1524              2.5244        0.2121            2.6002        0.2184        0.0000  9.1986
   1525              2.5239        0.2120            2.5967        0.2181        0.0000  9.1730
   1526              2.5303        0.2126            2.5979        0.2182        0.0000  9.1933
   1527              2.5281        0.2124            2.5992        0.2183        0.0000  9.1575
   1528              2.5269        0.2123            2.6010        0.2185        0.0000  9.2009
   1529              2.5254        0.2122            2.5968        0.2182        0.0000  9.1894
   1530              2.5299        0.2125            2.6029        0.2187        0.0000  9.1814
   1531              2.5253        0.2121            2.5995        0.2184        0.0000  9.2192
   1532              2.5256        0.2122            2.5991        0.2183        0.0000  9.1818
Stopping since valid_loss has not improved in the last 100 epochs.
Training completed in 16318.144165039062s
```

**Comment by nicoleyghu** [comment]
It seems that the training doesn't improve the error accuracy at all over 100 epochs. Please double-check the input and possibly run the examples from AmpTorch package to make sure the training is done properly.
